sympy/core/tests/test_args.py

```diff
@@ -468,8 +468,8 @@ def test_sympy__sets__sets__Complement():
 
 def test_sympy__sets__sets__SymmetricDifference():
     from sympy.sets.sets import FiniteSet, SymmetricDifference
-    assert _test_args(SymmetricDifference(FiniteSet(1, 2, 3), \
-           FiniteSet(2, 3, 4)))
+    assert _test_args(SymmetricDifference(FiniteSet(1, 2, 3),
```

Comment:
I agree that formatting can be done if the code is changed anyway; however, in this case
1) You aren't changing the code near these lines (though you do remove a superfluous backslash),
2) the new formatting is not an actual improvement (both ways to format the code are PEP8-compliant, but the changed formatting style is sometimes considered less good).

Strictly speaking for myself and not for the project, I'd recommend keeping subexpressions on one line as far as possible, and indent them by multiples of 4. Something like this:

```
assert _test_args(
    SymmetricDifference(FiniteSet(1, 2, 3), FiniteSet(2, 3, 4)))
```

Still, this isn't really a change related to your other work; I wouldn't -1 the PR if it is in, but I'd recommend keeping formatting away from "real work" so that people don't need to scroll over it when they're looking for what's in the PR description.